great_expectations/profile/user_configurable_profiler.py

```diff
@@ -67,6 +68,7 @@ def __init__(
         semantic_types_dict: Optional[Dict[str, List[str]]] = None,
         table_expectations_only: bool = False,
         value_set_threshold: str = "MANY",
+        data_context: DataContext = None,
```

Comment:
This cannot be none, because None is not supported by the ExpectationSuite constructor